src/group_theory/abelianization.lean

```diff
@@ -20,7 +20,7 @@ groups, which can be found in `algebra/category/Group/adjunctions`.
   commutator subgroup.
```

Comment:
Can you add `abelianization.map` and `mul_equiv.abelianization_congr` to the list here?